core/pipe/pipe.unit2.ts

```diff
@@ -0,0 +1,32 @@
+import {defaultPipeUnit} from './units/default.pipe.unit';
```

Comment:
why u need that? also there is no ts in core

Reply:
fixed